runtime/parachains/src/paras.rs

```diff
@@ -886,10 +888,24 @@ impl<T: Config> Pallet<T> {
 
 	/// Schedule a para to be cleaned up at the start of the next session.
 	///
-	/// Will return error if para is not a stable parachain or parathread.
+	/// Will return error if either is true:
+	///
+	/// - para is not a stable parachain or parathread.
```

Comment:
What is a stable parachain?

Reply:
a stable parachain or parathread is a parachain which `ParaLifecycles::<T>::get(&para)` is either `Parachain` or `Parathread`, and not e.g. `Onboarding` or other transitory state.

Do you think this should be clarified?

Reply:
Ahh okay, never heard this before. 

`para is not an on-boarded parachain or parathread.`

Would maybe have been more clear to me. 

Or you could use something like
`[`ParaLifecycle::is_stable`] returns `false` for para`

This way we would directly link to the particular type and make it more clear?

Reply:
Sure, good idea